src/modules/LoggedIn/Project/ProjectList.tsx

```diff
@@ -15,54 +15,61 @@ const ProjectList = () => {
-            <Card loading></Card>
-          </Col>
-          <Col span={6}>
-            <Card loading></Card>
-          </Col>
-          <Col span={6}>
-            <Card loading></Card>
-          </Col>
-          <Col span={6}>
-            <Card loading></Card>
-          </Col>
-        </>
-      ) : (
-        projectSync.projects.map(project => {
-          return (
-            <Col span={6} key={project._id}>
-              <Card
-                hoverable
-                actions={[
-                  <Popconfirm
-                    okButtonProps={{ loading: projectSync.isMutating }}
-                    title="Are you sure delete this project ?"
-                    onConfirm={() => confirm(project._id)}
-                    okText="Yes"
-                    cancelText="No"
-                  >
-                    <DeleteOutlined
-                      style={{
-                        fontSize: 20,
-                      }}
-                    />
-                  </Popconfirm>,
-                ]}
-              >
-                <Card.Meta
-                  title={project.name}
-                  description={project.description}
-                />
-              </Card>
+    <>
+      <Row style={{ marginTop: 16 }} gutter={[24, 16]}>
+        {projectSync.isValidating ? (
+          <>
+            <Col span={6}>
+              <Card loading></Card>
             </Col>
-          );
-        })
-      )}
-    </Row>
+            <Col span={6}>
+              <Card loading></Card>
+            </Col>
+            <Col span={6}>
+              <Card loading></Card>
+            </Col>
+            <Col span={6}>
+              <Card loading></Card>
+            </Col>
+          </>
+        ) : (
+          projectSync.projects.map(project => {
+            return (
+              <Col span={6} key={project._id}>
+                <Card
+                  hoverable
+                  actions={[
+                    <Popconfirm
+                      okButtonProps={{ loading: projectSync.isMutating }}
+                      title="Are you sure delete this project ?"
+                      onConfirm={() => confirm(project._id)}
+                      okText="Yes"
+                      cancelText="No"
+                    >
+                      <DeleteOutlined
+                        style={{
+                          fontSize: 20,
+                        }}
+                      />
+                    </Popconfirm>,
+                  ]}
+                >
+                  <Card.Meta
+                    title={project.name}
+                    description={project.description}
+                  />
+                </Card>
+              </Col>
+            );
+          })
+        )}
+      </Row>
+      <Layout style={{ padding: 50, alignItems: "center" }}>
+        <Row align="middle" justify="space-between">
+          <Button type="primary">Show More</Button>
+        </Row>
+      </Layout>
```

Comment:
Ini kyknya gak perlu component Layout deh, paddingnya bisa langsung ditaruh di component Row. Terus text dan type buttonnya disesuaikan sama figma ya. Di figma text buttonnya "Show more project" dan type buttonnya default, jadi typenya gak perlu ditulis gapapa, biar pake default. Jangan lupa, import2 yang gak dipake dihapus aja

```suggestion
 <Row style={{ padding: 50 }} align="middle" justify="space-between">
     <Button>Show More Project</Button>
 </Row>
```